packages/orga/src/timestamp.ts

```diff
@@ -20,7 +20,6 @@ ${close}\
   export const full = `^\\s*\
 (${single('begin')})\
 (?:--${single('end')})?\
-\\s*$\
```

Comment:
is this going to cause problems for us? so if I'm not mistaken, this allows anything comes after 1+ legit planning matching and will be ignored.

Reply:
Yes, that's right, but it allows the timestamp to be parsed correctly regardless of what comes after it.  I guess your point is that the ignored stuff won't make it into the tree, not even as plain text.  That's true, but shouldn't be after pull request #57 is merged since the rest is passed back to be further parsed.  I probably should have included it only in that pull request.  I can refactor the commit if you want.  Moreover, per the issue #55 it isn't showing up in the tree now with extra stuff at the end of the line.